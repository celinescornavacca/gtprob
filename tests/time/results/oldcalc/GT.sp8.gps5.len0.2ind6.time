0.29user 0.07system 0:00.49elapsed 75%CPU (0avgtext+0avgdata 30444maxresident)k
125inputs+0outputs (0major+12788minor)pagefaults 0swaps
