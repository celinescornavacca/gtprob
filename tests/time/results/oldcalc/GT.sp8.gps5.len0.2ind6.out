-71.477252784
